### DISCUSSION TITLE
Pivot Table v2,  suggest that separate the options on showing Total/Subtotal values

### DISCUSSION THREAD

**Discussion by xwu2021** [apache/superset#20234] [Ideas]
## Description
When I use Pivot Table v2 to display different classes of datas,  subtotal function is wonderful. But,  at the same time, column "Total"  is also calculated. In most of situations, this 'total' column doesn't make sense.
So how about to separate the options on showing Total/Subtotal values.

There is an example and my suggestion:
![image](https://user-images.githubusercontent.com/81284618/171125362-8bb90f31-6fae-4071-80b8-0e03f4605362.png)

**Comment by adimyth** [comment]
Any updates on this one?

@xwu2021 were you able to solve for this?

**Comment by shompp** [comment]
totally agree, in many cases it doesn't make any sense, please add feature to only show subtotals

**Comment by michael-s-molina** [comment]
I submitted a PR to implement this feature: https://github.com/apache/superset/pull/24960

**Comment by flame-brk** [comment]
we realised what for our use case such implementation isn't enough
more flexible approach can be block of checkboxes, to pick only dimensions desired for subtotal (similar to excel pivot table)

**Comment by george040875** [comment]
I had resolved it by css:
th.pvtTotalLabel{  display: none; }
td.pvtTotal{ display: none; }

**Comment by cxbxtx** [comment]
You can hide any column using a css-template.
For your case try this css:
```
table.pvtTable thead tr:nth-child(1) th.pvtTotalLabel,
table.pvtTable tbody tr.pvtRowTotals td.pvtGrandTotal.pvtRowTotal,
table.pvtTable tbody tr td[role="gridcell"][class="pvtTotal"] {
    display: none;
}
```
